examples/debian_snapshot/test_linux_amd64.yaml

```diff
@@ -18,6 +18,7 @@ commandTests:
       - ncurses-base/now 6\.2\+20201114-2\+deb11u2 all \[installed,local\]
       - perl/now 5\.32\.1-4\+deb11u3 amd64 \[installed,local\]
       - tzdata/now 2024a-0\+deb11u1 all \[installed,local\]
+      - r-base-core/now 4.4.0-2~bullseyecran.0 i386 \[installed,local\]
```

Comment:
`i386` is the wrong architecture here, this is caused by a bug in `_parse_package_index`, see my comments before in this commit, in [`apt/private/package_index.bzl`](https://github.com/GoogleContainerTools/rules_distroless/pull/55/files/b84a604cd93533afd2c222cca03f4f63a35df85d#diff-2c0a36dc8512cca883457e90c6bd82fb340b10d91d4f0127eb9e5aae38e68b11).